locale/en/docs/guides/backpressuring-in-streams.md

```diff
@@ -88,6 +88,68 @@ a chunk of data were to fail to be properly received, the `Readable` source or
 properly destroy all the streams in a pipeline if one of them fails or closes,
 and is a must have in this case!
 
+As for Node 10.0.0 or later version, [`pipeline`][] is introduced to replace for
+[`pump`][]. This is a module method to pipe between streams forwarding errors
+and properly cleaning up and provide a callback when the pipeline is complete.
+
+You can use it like this following:
+
+```javascript
+const { pipeline } = require('stream');
+const fs = require('fs');
+const zlib = require('zlib');
+
+// Use the pipeline API to easily pipe a series of streams
+// together and get notified when the pipeline is fully done.
+// A pipeline to gzip a potentially huge tar file efficiently:
+
+pipeline(
+  fs.createReadStream('The.Matrix.1080p.mkv'),
+  zlib.createGzip(),
+  fs.createWriteStream('The.Matrix.1080p.mkv.gz'),
+  (err) => {
+    if (err) {
+      console.error('Pipeline failed', err);
+    } else {
+      console.log('Pipeline succeeded');
+    }
+  }
+);
+```
+or if you wanna `async` with `await`, you can use it wrapped by [`promisify`][]:
+
+```javascript
+const stream = require('stream');
+const fs = require('fs');
+const zlib = require('zlib');
+
+const pipeline = util.promisify(stream.pipeline);
+
+async function run() {
+  await pipeline(
+     fs.createReadStream('The.Matrix.1080p.mkv'),
+     zlib.createGzip(),
+     fs.createWriteStream('The.Matrix.1080p.mkv.gz'),
+  );
+}
+
+// Either of the following ways you can call the `run()`:
+
+// Way 1: You can use this to catch exceptions in async mode.
+run().catch((err) => console.log('Pipeline failed', err));
```

Comment:
I feel this is redundant, maybe just having the following code snippet is enough:

```js
async function run() {
  try {
    await pipeline(
       fs.createReadStream('The.Matrix.1080p.mkv'),
       zlib.createGzip(),
       fs.createWriteStream('The.Matrix.1080p.mkv.gz'),
    );
  } catch(err) {
    console.error('Pipeline failed', err);
  }
}
```

Reply:
Fixed